src/install-peerdeps.js

```diff
@@ -98,6 +100,27 @@ function spawnInstall(command, args) {
   });
 }
 
+/**
+ * Read package file file from disk.
+ * Returns undefined if no package file exists.
+ */
+function readPackageFile() {
```

Comment:
Could you add a test for this function in `install-peerdeps.test.js` under the `getPackageData` test?

Reply:
Done, see https://github.com/ashleydavis/install-peerdeps/commit/a78bb1f5af3908a03ae605bd29737b98beb85db1

Reply:
Thanks, I'll take a look!